src/draw/vglite/lv_vglite_utils.c

```diff
@@ -0,0 +1,539 @@
+/**
+ * @file lv_vglite_utils.c
+ *
+ */
+
+/**
+ * Copyright 2022-2024 NXP
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_vglite_utils.h"
+#include "lv_vglite_path.h"
+
+
+#if LV_USE_DRAW_VGLITE
+#include "lv_vglite_buf.h"
+
+#include "../../core/lv_refr.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ *  STATIC PROTOTYPES
+ **********************/
+
+/**********************
+ *  STATIC VARIABLES
+ **********************/
+
+#if LV_USE_VGLITE_DRAW_ASYNC
+    static volatile bool _cmd_buf_flushed = false;
+#endif
+
+/**********************
+ *      MACROS
+ **********************/
+
+/**********************
+ *   GLOBAL FUNCTIONS
+ **********************/
+
+const char * vglite_error_to_string(vg_lite_error_t error)
+{
+    switch(error) {
+            ENUM_TO_STRING(VG_LITE_SUCCESS);
+            ENUM_TO_STRING(VG_LITE_INVALID_ARGUMENT);
+            ENUM_TO_STRING(VG_LITE_OUT_OF_MEMORY);
+            ENUM_TO_STRING(VG_LITE_NO_CONTEXT);
+            ENUM_TO_STRING(VG_LITE_TIMEOUT);
+            ENUM_TO_STRING(VG_LITE_OUT_OF_RESOURCES);
+            ENUM_TO_STRING(VG_LITE_GENERIC_IO);
+            ENUM_TO_STRING(VG_LITE_NOT_SUPPORT);
+            ENUM_TO_STRING(VG_LITE_ALREADY_EXISTS);
+            ENUM_TO_STRING(VG_LITE_NOT_ALIGNED);
+            ENUM_TO_STRING(VG_LITE_FLEXA_TIME_OUT);
+            ENUM_TO_STRING(VG_LITE_FLEXA_HANDSHAKE_FAIL);
+        default:
+            break;
+    }
+
+    return "VG_LITE_UKNOWN_ERROR";
+}
+
+void vglite_dump_info(void)
```

Comment:
Same as previous comment, it should be built when `LV_USE_VGLITE_DEBUG` is enabled.